ci/create_codewind_index.py

```diff
@@ -14,7 +14,7 @@
 
 for file in os.listdir(assets_dir):
     if fnmatch.fnmatch(file, '*index.yaml'):
-        with open(assets_dir + file, 'r') as yamlFile, open(assets_dir + os.path.splitext(file)[0] + ".json", 'w') as jsonFile:
+        with open(assets_dir + file, 'r') as yamlFile, open(assets_dir + os.path.splitext(file)[0] + ".json", 'wb') as jsonFile:
```

Comment:
Why did you change the "w" to "wb" here? This will always be writing a `.json` file which is a text file

Reply:
Without it, you'll get `TypeError: write() argument must be str, not bytes` error. There might be other ways around it but it's related to a change between Python2 and 3 and how it handles strings.

Reply:
Yes, I understand this now, thanks